### ISSUE TITLE
Bisq Maintainer

### ISSUE THREAD

**Issue by cbeams** [bisq-network/roles#63]
Docs: https://bisq.wiki/Bisq_Maintainer
Team: @bisq-network/bisq-maintainers

**Comment by cbeams** [comment]
I've assigned @ripcurlx to this role, reflecting the work that he is already doing on Bisq's UI. This is still a role that we'd like additional developers to take on, though. There is plenty of work to be done!

**Comment by cbeams** [comment]
I've renamed this role from JavaFX Developer => Bisq Desktop Maintainer. @ripcurlx, as the primary role owner here, please report monthly as you see fit.

**Comment by ripcurlx** [comment]
## 2018.06 report
Reviewing and merging of PRs and release preparations for v0.7.1

/cc bisq-network/compensation#84

**Comment by cbeams** [comment]
## 2018.06 report

Nothing to report. Note that I plan to remove myself from the set of desktop maintainers.

/cc bisq-network/compensation#89

**Comment by cbeams** [comment]
@ripcurlx wrote

> Reviewing and merging of PRs 

I think it's important that we account separately for maintainer work and PR review work. I've written about this in the WIP Roles doc, see https://deploy-preview-46--bisq-docs.netlify.com/roles.html#maintainer. I'm +1'ing your compensation request as-is, but something to keep in mind as you review that doc.

**Comment by cbeams** [comment]
I've updated the @bisq-network/desktop-maintainers team and the description of this issue to reflect that @ManfredKarrer and @ripcurlx are Bisq Desktop maintainers now, and that I've removed myself.

**Comment by ripcurlx** [comment]
## 2018.08 report

Release of [v0.8.0](https://github.com/bisq-network/bisq-desktop/releases/tag/v0.8.0)

/cc bisq-network/compensation#109

**Comment by cbeams** [comment]
Renamed to "Bisq Maintainer" to reflect the monorepo reconsolidation in bisq-network/bisq#1650 and added @cbeams as a member of @bisq-network/bisq-maintainers (which has also been renamed) to reflect his maintainer role over the `assets` subproject.

**Comment by ripcurlx** [comment]
## 2018.09 report

This month we reconsolidated all code repositories into one mono repo to make it easier for developers to work across modules and switched to Java 10. No release this month.

/cc bisq-network/compensation#126

**Comment by ripcurlx** [comment]
## 2018.10 report

We've created a [release branch for 0.9.0](https://github.com/bisq-network/bisq/tree/release-candidate-0.9.0) this time before merging to master to make it easier to cooperate and developer test the work happening in the redesign and DAO in parallel.

/cc bisq-network/compensation#148

**Comment by ripcurlx** [comment]
## 2018.11 report

We've been working on and syncing the [release branch for 0.9.0](https://github.com/bisq-network/bisq/tree/release-candidate-0.9.0) to make it easier to cooperate and developer test the work happening in the redesign and DAO in parallel. This month also release preparations and testing for v0.9.0 is part of my Desktop Maintainer contribution.

/cc bisq-network/compensation#177

**Comment by ripcurlx** [comment]
## 2018.12 report

We had a busy December with two releases ([v0.9.0](https://github.com/bisq-network/bisq/releases/tag/v0.9.0) and [v0.9.1](https://github.com/bisq-network/bisq/releases/tag/v0.9.1)). 
Release preparations and testing is part of my Desktop Maintainer contribution this month.

/cc bisq-network/compensation#195

**Comment by ripcurlx** [comment]
## 2019.01 report

We had again a busy month with two releases ([v0.9.2](https://github.com/bisq-network/bisq/releases/tag/v0.9.2) and [v0.9.3](https://github.com/bisq-network/bisq/releases/tag/v0.9.3)). We also added the [stale app](https://github.com/probot/stale) to help cleaning up our issue repository. It will mark issues that are inactive for 90 days as stale and close it without activity 7 days later. Excluded from becoming stale are issues with following labels:
- a:bug
 - re:security
- re:privacy
- re:Tor
- in:dao
- $BSQ bounty
- good first issue
- Epic

Release preparations and testing is part of my Desktop Maintainer contribution this month.

/cc bisq-network/compensation#206

**Comment by ripcurlx** [comment]
## 2019.02 report

We released [v0.9.4](https://github.com/bisq-network/bisq/releases/tag/v0.9.4).
Release preparations and testing is part of my Desktop Maintainer contribution this month.

/cc bisq-network/compensation#224

**Comment by ripcurlx** [comment]
## 2019.03 report

We released [v0.9.5](https://github.com/bisq-network/bisq/releases/tag/v0.9.5).
Release preparations and testing is part of my Desktop Maintainer contribution this month.

/cc bisq-network/compensation#247

**Comment by ripcurlx** [comment]
## 2019.04 report

We released [v1.0.0](https://github.com/bisq-network/bisq/releases/tag/v1.0.0) and [v1.0.1](https://github.com/bisq-network/bisq/releases/tag/v1.0.1).
Release preparations and testing is part of my Desktop Maintainer contribution this month.

/cc bisq-network/compensation#269

**Comment by sqrrm** [comment]
## Cycle 2 report
Took over as secondary Bisq maintainer. I have merged PRs and done minor git admin.

https://github.com/bisq-network/compensation/issues/293

**Comment by ripcurlx** [comment]
## Cycle 2 report

We released [v1.1.2](https://github.com/bisq-network/bisq/releases/tag/v1.1.2).

Release preparations and testing is part of my Desktop Maintainer contribution for this cycle.

**Comment by ripcurlx** [comment]
## Cycle 3 report

No release this cycle as we are still working on the account signing feature. Probably we'll release one more version before this feature as a couple of PRs already piled up.

**Comment by sqrrm** [comment]
## Cycle 3 report

All maintainer work is included under other posts for compensation, as appropriate.

https://github.com/bisq-network/compensation/issues/311

**Comment by bisq-github-admin-3** [comment]
I just added @freimair as a Bisq maintainer to increase the issue and PR throughput.

**Comment by sqrrm** [comment]
## Cycle 4 report

All maintainer work is included under other posts for compensation, as appropriate.

@ripcurlx handled releases, helped with testing and preparation.

bisq-network/compensation#325

**Comment by ripcurlx** [comment]
## Cycle 4 report

We released [v1.1.3](https://github.com/bisq-network/bisq/releases/tag/v1.1.3) and [v1.1.4](https://github.com/bisq-network/bisq/releases/tag/v1.1.4).

Release preparations and testing is part of my Desktop Maintainer contribution for this cycle.

/cc https://github.com/bisq-network/compensation/issues/269

**Comment by sqrrm** [comment]
## Cycle 5 report

All maintainer work is included under other posts for compensation, as appropriate.

Didn't help much with the latest release, thanks to @ripcurlx for all the work

https://github.com/bisq-network/compensation/issues/356

**Comment by ripcurlx** [comment]
## Cycle 6 report

We released [v1.1.6](https://github.com/bisq-network/bisq/releases/tag/v1.1.6) and [v1.1.7](https://github.com/bisq-network/bisq/releases/tag/v1.1.7).

Release preparations and testing is part of my Desktop Maintainer contribution for this cycle.

/cc https://github.com/bisq-network/compensation/issues/369

**Comment by sqrrm** [comment]
## Cycle 6 report

All maintainer work is included under other posts for compensation, as appropriate.

The other maintainers have taken most of the work this month

https://github.com/bisq-network/compensation/issues/375

**Comment by ripcurlx** [comment]
## Cycle 7 report

We released [v1.2.0](https://github.com/bisq-network/bisq/releases/tag/v1.2.0), [v1.2.1](https://github.com/bisq-network/bisq/releases/tag/v1.2.1), [v1.2.2](https://github.com/bisq-network/bisq/releases/tag/v1.2.2) and [v1.2.3](https://github.com/bisq-network/bisq/releases/tag/v1.2.3).

Release preparations and testing is part of my Desktop Maintainer contribution for this cycle.

/cc https://github.com/bisq-network/compensation/issues/401

**Comment by sqrrm** [comment]
## Cycle 7 report
All related work, such as merges and release preparation, has been documented elsewhere, 

https://github.com/bisq-network/compensation/issues/403

**Comment by ripcurlx** [comment]
## Cycle 8 report

We released [v1.2.4](https://github.com/bisq-network/bisq/releases/tag/v1.2.4).

Release preparations is part of my Desktop Maintainer contribution for this cycle.

/cc https://github.com/bisq-network/compensation/issues/439

**Comment by sqrrm** [comment]
## Cycle 8 report

Not much done, been away. Some merges and didn't help with the release.

bisq-network/compensation#441

**Comment by sqrrm** [comment]
## Cycle 9 report

Quite a bit of work on reviews and merges, as documented in the compensation request.

Helped out with release testing.

bisq-network/compensation#461

**Comment by ripcurlx** [comment]
## Cycle 9 report

We released [v1.2.5](https://github.com/bisq-network/bisq/releases/tag/v1.2.5).

Release preparations is part of my Desktop Maintainer contribution for this cycle.

/cc https://github.com/bisq-network/compensation/issues/466

**Comment by sqrrm** [comment]
## Cycle 10 report

Things have changed a bit with the new stricter structure stewarded by strongman Chris (really would've been better had it been Steve for my alliteration). We're trying to focus the reviews on prioritized work. Less fluff all around, only place left for fluffy fearless phrasing is the Bisq maintainer report.

bisq-network/compensation#488

**Comment by ripcurlx** [comment]
## Cycle 10 report

So much peer pressure by role peers to create more fluffy fearless phasing this time. Unfortunately not part of the Austrian mentality, so no fluff for me this time  😜

We released [v1.2.6](https://github.com/bisq-network/bisq/releases/tag/v1.2.6). To reduce unnecessary review time I've added [Codacy](https://app.codacy.com/gh/bisq-network/bisq/dashboard) now to the Bisq repository as well. It enforces certain development guidelines and will be adapted over time until a PR review will only be about the code changed and not about the lack of formatting, unused imports or similar stuff.

Release preparations are NOT part of my Desktop Maintainer contribution for this cycle, as I'll add it separately to the compensation request to improve transparency.

/cc https://github.com/bisq-network/compensation/issues/493

**Comment by sqrrm** [comment]
## Cycle 12 report

Some introspection is warranted after the failures of reviews regarding the recent security issues. I see it as a failure of all parts of the Bisq DAO, and for this role it would be in the review letting this kind of issue slip through. Should we spend more time on reviews? Is there just a lack of eyes on code being merged? Do we have a structural flaw or just an issue of execution?

After we have a post mortem a discussion around these questions and much more would be useful.

https://github.com/bisq-network/compensation/issues/536

**Comment by ripcurlx** [comment]
## Cycle 12 report

We released [v1.2.9](https://github.com/bisq-network/bisq/releases/tag/v1.2.9), [v1.3.0](https://github.com/bisq-network/bisq/releases/tag/v1.3.0) and [v1.3.1](https://github.com/bisq-network/bisq/releases/tag/v1.3.1). I do agree with @sqrrm that we need to do a proper review why this security vulnerability happened and how we can prevent it in the future. Also our release, hotfix release cadence needs to be ended. We are trying something out already for v1.3.2, but it needs to be tested, formalized and properly executed from now on.

/cc https://github.com/bisq-network/compensation/issues/542

**Comment by sqrrm** [comment]
## Cycle 13 report

I've taken over the admin duties that @ripcurlx usually handle. Acking CRs, more vigilant eye on PRs (which I should be doing anyway, but haven't been great at) and trying to keep the dev priorities in order. While I'm not excelling I'm spending an inordinate amount of time on stuff, just stuff. Hope for a swift return of @ripcurlx to handle things more efficiently.

bisq-network/compensation#564

**Comment by sqrrm** [comment]
## Cycle 14 report

Spent more time on PR overview, meetings and planning than my normal schedule. @ripcurlx will likely be back next week, very good news indeed.

bisq-network/compensation#597

**Comment by sqrrm** [comment]
## Cycle 15 report

Didn't do much specific maintaining. Still doing reviews but not as much admin.

https://github.com/bisq-network/compensation/issues/623

**Comment by ripcurlx** [comment]
## Cycle 13 -15 report

We released [v1.3.2](https://github.com/bisq-network/bisq/releases/tag/v1.3.2), [v1.3.3](https://github.com/bisq-network/bisq/releases/tag/v1.3.3), [v1.3.4](https://github.com/bisq-network/bisq/releases/tag/v1.3.4), [v1.3.5](https://github.com/bisq-network/bisq/releases/tag/v1.3.5) and [v1.3.6](https://github.com/bisq-network/bisq/releases/tag/v1.3.6).

@sqrrm Took over this role mainly for cycle 13 and 14 and I only did the bare minimum during that time (Thanks!). In cycle 15 I took again over the role and am still trying to catch up with everything that stacked up during that time. Still there is light at the end of the tunnel and we started to re-pay victims by the BTC fees and continue to ship great additions to the Bisq client.

/cc https://github.com/bisq-network/compensation/issues/625

**Comment by ripcurlx** [comment]
## Cycle 16 report

We released [v1.3.7](https://github.com/bisq-network/bisq/releases/tag/v1.3.7).

/cc https://github.com/bisq-network/compensation/issues/652

**Comment by sqrrm** [comment]
## Cycle 16 report

Been a bit more with reviews and other admin details this month. Hope to see codacy in reasonable shape eventually

bisq-network/compensation#646

**Comment by sqrrm** [comment]
## Cycle 17 report
A lot of activity this month. Many merge issues due to Codacy. It's a pity that it might be causing more trouble than benefit. I'm suggesting that we disable the enforced pass to allow merging when we don't like what it's saying.

https://github.com/bisq-network/compensation/issues/667

**Comment by ripcurlx** [comment]
> A lot of activity this month. Many merge issues due to Codacy. It's a pity that it might be causing more trouble than benefit. I'm suggesting that we disable the enforced pass to allow merging when we don't like what it's saying.
> 
> [bisq-network/compensation#667](https://github.com/bisq-network/compensation/issues/667)

I agree that it caused troubles in the past, but I think it is mainly a matter of configuration and tuning until it works as we want it to be. But yes, we can remove the enforced pass to allow merging until we have reached that point.

![Bildschirmfoto 2020-09-22 um 15 38 10](https://user-images.githubusercontent.com/170962/93889622-b60b6100-fce9-11ea-918e-baa296483644.png)

Actually it does seem to be configured that way already.

**Comment by ripcurlx** [comment]
## Cycle 17 report

We pre-released [v1.3.8](https://github.com/bisq-network/bisq/releases/tag/v1.3.8) and publicly released [v1.3.9](https://github.com/bisq-network/bisq/releases/tag/v1.3.9).

/cc https://github.com/bisq-network/compensation/issues/671

**Comment by sqrrm** [comment]
## Cycle 18 report

Almost 100% review this cycle. That includes talks on design and testing. We managed to release most of it after some minor issues.

https://github.com/bisq-network/compensation/issues/693

**Comment by ripcurlx** [comment]
## Cycle 18 report

We pre-released [v1.4.0](https://github.com/bisq-network/bisq/releases/tag/v1.4.0), [v1.4.1](https://github.com/bisq-network/bisq/releases/tag/v1.4.1) and publicly released [v1.4.2](https://github.com/bisq-network/bisq/releases/tag/v1.4.2).
Besides taking care of issues together with @cd2357, who got the triage permission for the Bisq repository recently, I mainly worked on our upcoming build system to be able to switch to a LTS Java version and most up-to-date JavaFX version.

/cc https://github.com/bisq-network/compensation/issues/704

**Comment by sqrrm** [comment]
## Cycle 19 report

There was a lot of work to get 1.5.0 released with all the issues we had with 1.4.2. So far 1.5.0 seems to be at least not worse than 1.4.2 which is a good thing. There are still quite a lot of support cases lingering from 1.4.2 so I would consider stronger incentives for users to upgrade if we don't get a good uptake.

https://github.com/bisq-network/compensation/issues/721

**Comment by ripcurlx** [comment]
## Cycle 19 report

We released [v1.5.0](https://github.com/bisq-network/bisq/releases/tag/v1.5.0) without a pre-release phase as it is not possible to trade with older versions because of the trade protocol changes. Also we wanted to get the bug fixes introduced in v1.4.2 out there as soon as possible.  We do have a [70%+ adoption rate of v1.5.0](https://monitor.bisq.network/d/dEVJDduWz/key-performance-indicators?orgId=1&from=now-2d&to=now&fullscreen&panelId=3) already, so I hope support cases should decrease soon. I do check now on a daily basis all GitHub notifications to react in-time on issues and PR changes to prevent unnecessary blocks from maintainers side.

/cc https://github.com/bisq-network/compensation/issues/723

**Comment by ripcurlx** [comment]
## Cycle 20 report

Although we fixed with [v1.5.1](https://github.com/bisq-network/bisq/releases/tag/v1.5.1) already most of the issues experienced in v1.5.0,  [v1.5.2](https://github.com/bisq-network/bisq/releases/tag/v1.5.2) shipped as a regular release should finally bring some calm over the holidays into mediation and support (hopefully).

I continue to check on a daily basis all GitHub notifications to react in-time on issues and PR changes to prevent unnecessary blocks from maintainers side. Over the holidays I wasn't able to deliver on this, but I'll try my best to clean out my notifications log today.

[EDIT]
I did forget to mention that I had to renew my Code Signing Certificate for Windows for €91.55 (~$111). I'll add this to the desktop maintainer amount.
[/EDIT]

/cc https://github.com/bisq-network/compensation/issues/746

**Comment by sqrrm** [comment]
## Cycle 20 report
A fair few issues going on from the previous releases which added to the review and merge workload. Reviews are in general reported separately but there is an overhead of having to monitor PRs and handling the releases.

https://github.com/bisq-network/compensation/issues/747

**Comment by sqrrm** [comment]
## Cycle 21 report

Voting in last cycle failed which so far caused less problems than I would've expected. The DAO code worked as expected and contributors have been very good about it.

https://github.com/bisq-network/compensation/issues/773

**Comment by ripcurlx** [comment]
## Cycle 21 report

We released [v1.5.3](https://github.com/bisq-network/bisq/releases/tag/v1.5.3) and [v1.5.4](https://github.com/bisq-network/bisq/releases/tag/v1.5.4) as a hotfix release as we experienced behavior that looked like a DoS attack. Since those two releases everything is working again (except the TOR related issues every now and then, which are based on a real DoS attack).

/cc https://github.com/bisq-network/compensation/issues/758

**Comment by ripcurlx** [comment]
## Cycle 22 report

We released [v1.5.5](https://github.com/bisq-network/bisq/releases/tag/v1.5.5), [v1.5.6 of ](https://github.com/bisq-network/bisq/releases/tag/v1.5.6), [v1.5.7](https://github.com/bisq-network/bisq/releases/tag/v1.5.7), [v1.5.8](https://github.com/bisq-network/bisq/releases/tag/v1.5.8), and [v1.5.9](https://github.com/bisq-network/bisq/releases/tag/v1.5.9). v1.5.6 was a hotfix that fixed startup issues for limit offers, v1.5.8 was a follow-up release introducing dynamic min tx fee powered by mempool.space and v1.5.9 a hotfix for a mailbox message startup issue.

/cc https://github.com/bisq-network/compensation/issues/795

**Comment by sqrrm** [comment]
## Cycle 22 report

I was unwell large part of this cycle. Did minimal review and merge, not requesting anything for the maintainer role.

https://github.com/bisq-network/compensation/issues/788

**Comment by ripcurlx** [comment]
## Cycle 23 report

We released [v1.6.0](https://github.com/bisq-network/bisq/releases/tag/v1.6.0), [v1.6.1 of ](https://github.com/bisq-network/bisq/releases/tag/v1.6.1) and [v1.6.2](https://github.com/bisq-network/bisq/releases/tag/v1.6.2). v1.6.1 was a follow-up release after a v1.6.0 pre-release and v1.6.2 a hotfix release as an issue with SPV resync was introduced in v1.6.0.

/cc https://github.com/bisq-network/compensation/issues/813

**Comment by sqrrm** [comment]
## Cycle 23 report

I haven't done all that much on this role except being moral support for @ripcurlx 

https://github.com/bisq-network/compensation/issues/817

**Comment by sqrrm** [comment]
## Cycle 24 report

Still doing limited maintainer work

https://github.com/bisq-network/compensation/issues/838

**Comment by ripcurlx** [comment]
## Cycle 24 report

We released [v1.6.3](https://github.com/bisq-network/bisq/releases/tag/v1.6.3) and [v1.6.4](https://github.com/bisq-network/bisq/releases/tag/v1.6.4). v1.6.4 was a follow-up release after a v1.6.3 pre-release as we wanted to filter out one infrastructure change and a graceful restart if banned seednode addresses are misconfigured. This was triggered by myself after I published a seednode address to be banned that had an incorrect format. As there is no validation in the filter form, but there is one when the bisq.properties file is read on startup this prevented all nodes that stopped or restarted within a time window of three hours to start. The workaround to start again was to remove the bannedSeednodes line from the bisq.properties file. To prevent this in the future I'll add validation to the filter fields and already removed incorrect configured nodes when read in v1.6.4.

This two releases also include a completely new way to create the installers enabling us to move to a LTS Java version again (11) and also to be able to update JavaFX form now on again.

/cc https://github.com/bisq-network/compensation/issues/846

**Comment by sqrrm** [comment]
## Cycle 25 report

Still doing minimal maintainer work.

https://github.com/bisq-network/compensation/issues/855

**Comment by ripcurlx** [comment]
## Cycle 25 report

We released [v1.6.5](https://github.com/bisq-network/bisq/releases/tag/v1.6.5) - no hotfix this time 🥳 
The new build pipeline works pretty fine, except for one [Java Heap Space Error](https://github.com/bisq-network/bisq/issues/5550) that is popping up since v1.6.4.

I had to move Travis from https://travis-ci.org to https://travis-ci.com with the downside that even with the free OS plan we have to ask for 25k new free build credits every now and then. Thanks to https://github.com/maxim-belkin we already have a GitHub action in place that verifies nearly everything the current Travis build does. Probably worth to add the last bit as well and get rid of Travis to reduce future maintenance issues.

I also prepared an [additional CodeQL GitHub action](https://github.com/bisq-network/bisq/pull/5554) to run security checks over the code base as well. I also plan to add a security policy to the repository soon.

/cc https://github.com/bisq-network/compensation/issues/860

**Comment by sqrrm** [comment]
## Cycle 26 report

All release work has been reported in other posts, @ripcurlx is keeping up the more organized maintenance work.

https://github.com/bisq-network/compensation/issues/874

**Comment by ripcurlx** [comment]
## Cycle 26 report

We released [v1.7.0](https://github.com/bisq-network/bisq/releases/tag/v1.7.0) which was a hard fork because of a privacy issue reported by @woodser.  This release needed in-depth testing because of different upgrade scenarios and was deployed without a pre-release version. So far we only saw issue during dispute handling which was addressed in pre-release [v1.7.1](https://github.com/bisq-network/bisq/releases/tag/v1.7.1). Our users started to see lots of DAO not in-sync popups which we weren't aware of before. This is still in investigation to find the exact root cause. We continue to see Java heap errors which should be addressed with version v1.7.2 which will be released soon. Because of vacation I won't be very active over the next three weeks, but I'll try to keep up with my duties as good as possible.

/cc https://github.com/bisq-network/compensation/issues/882

**Comment by ripcurlx** [comment]
## Cycle 27 report

We released [v1.7.2](https://github.com/bisq-network/bisq/releases/tag/v1.7.2) which mainly focused to reduce memory consumption and fixes a couple of minor issues that were introduced in the v1.7.0 hardfork. I wasn't very active in this cycle because of vacation.

/cc https://github.com/bisq-network/compensation/issues/904

**Comment by ripcurlx** [comment]
## Cycle 28 report

We released a [v1.7.3] (https://github.com/bisq-network/bisq/releases/tag/v1.7.3) vacation release which mainly contained minor bug fixes and UI improvements. I returned to attend to my duties as maintainer and was mainly focusing on the Tor v3 support issue this cycle.

/cc https://github.com/bisq-network/compensation/issues/904

**Comment by ripcurlx** [comment]
## Cycle 29 report

We released [v1.7.4] (https://github.com/bisq-network/bisq/releases/tag/v1.7.4) containing the Tor v3 support. So far the release went fine and most of the network already upgraded to Tor v3 addresses as well.

/cc https://github.com/bisq-network/compensation/issues/931

**Comment by ripcurlx** [comment]
## Cycle 30 report

We released [v1.7.5](https://github.com/bisq-network/bisq/releases/tag/v1.7.5) introducing 13 new payment methods and moving all remaining nodes to Tor v3 addresses.

/cc https://github.com/bisq-network/compensation/issues/939

**Comment by ripcurlx** [comment]
## Cycle 31 report

We released [v1.8.0](https://github.com/bisq-network/bisq/releases/tag/v1.8.0) introducing BSQ swaps and major DAO parsing improvements. We had to increase the mandatory version to v1.8.0 as a compatibility issue, when trading with an older version with a specific payment method, surfaced. ATM we haven't set any difficulty for BSQ swaps and are also not enforcing new trading fee validations shipped with v1.8.0. This will change within the next couple of days. As part of the Windows release process I also had to renew my code signing certificate over GoGetSSL (€91,71).

/cc https://github.com/bisq-network/compensation/issues/966

**Comment by ripcurlx** [comment]
## Cycle 32 & 33 report

We released [v1.8.1](https://github.com/bisq-network/bisq/releases/tag/v1.8.1) and [v1.8.2](https://github.com/bisq-network/bisq/releases/tag/v1.8.2) improving mediation, extending the API capabilities and fixing lots of smaller issues across the board. I'm a little less active than in previous cycles because of personal reasons, which should change to the better soon again.

/cc https://github.com/bisq-network/compensation/issues/1016

**Comment by ripcurlx** [comment]
## Cycle 34 report

We released [v1.8.3](https://github.com/bisq-network/bisq/releases/tag/v1.8.3) and [v1.8.4](https://github.com/bisq-network/bisq/releases/tag/v1.8.4) improving privacy by automatically deleting past trade data. Besides that lots of small UI improvements and bug fixes.

Improvements


/cc https://github.com/bisq-network/compensation/issues/1041

**Comment by ripcurlx** [comment]
## Cycle 35 report

We didn't release in this cycle as v1.9.0 development took longer than expected.

/cc https://github.com/bisq-network/compensation/issues/1060

**Comment by ripcurlx** [comment]
## Cycle 36 report

We released [v1.9.0](https://github.com/bisq-network/bisq/releases/tag/v1.9.0) (pre-release) and [v1.9.1](https://github.com/bisq-network/bisq/releases/tag/v1.9.1) (public release) introducing the Bisq API and featuring a new navigation structure. 

/cc https://github.com/bisq-network/compensation/issues/1088

**Comment by ripcurlx** [comment]
## Cycle 37 report

We released with [v1.9.2](https://github.com/bisq-network/bisq/releases/tag/v1.9.2) a hotfix release based on v1.9.1 including the latest official Tor binary at the time (v0.4.7.7). I blocked one onion address (nlpqkpun5alxgmtfcworaunumrcbpupha6l6perphk45ki7fd736ueyd.onion) and related accounts from trading because of chargeback attempts based on suggestion from @pazza83 and @leo816.

/cc https://github.com/bisq-network/compensation/issues/1088

**Comment by ripcurlx** [comment]
## Cycle 38 report

We released with [v1.9.3](https://github.com/bisq-network/bisq/releases/tag/v1.9.3) a pre-release and [v1.9.4](https://github.com/bisq-network/bisq/releases/tag/v1.9.4) as the public release to improve XMR trading and Tor connectivity.

/cc https://github.com/bisq-network/compensation/issues/1154

**Comment by ripcurlx** [comment]
## Cycle 39 report

We released with [v1.9.5](https://github.com/bisq-network/bisq/releases/tag/v1.9.5) a new P2P status indicator and adding Bitcoin Core v23 compatibility.

/cc https://github.com/bisq-network/compensation/issues/1154

**Comment by ripcurlx** [comment]
## Cycle 40 report

We didn't release a new version last cycle, as we wait for a couple of important changes.

/cc https://github.com/bisq-network/compensation/issues/1154

**Comment by ripcurlx** [comment]
## Cycle 41 report

We released with [v1.9.6](https://github.com/bisq-network/bisq/releases/tag/v1.9.6) updating Tor and removing macOS notarization and Windows code signing.

/cc https://github.com/bisq-network/compensation/issues/1184

**Comment by ripcurlx** [comment]
## Cycle 42 report

Because of major changes in the codebase the v1.9.7 release was delayed.

/cc https://github.com/bisq-network/compensation/issues/1197

**Comment by ripcurlx** [comment]
## Cycle 43 report

We released with the [v1.9.7](https://github.com/bisq-network/bisq/releases/tag/v1.9.7) pre-release and the follow-up public release of [v1.9.8](https://github.com/bisq-network/bisq/releases/tag/v1.9.8) many improvements under the hood to improve resilience of the client and push further the decentralization of Bisq. With this release we also introduced new signers for the public release. Following is the signed public key of @alejandrogarcia83 with my private key. 

```
-----BEGIN PGP PUBLIC KEY BLOCK-----
Comment: CB36 D7D2 EBB2 E35D 9B75  500B CD5D C1C5 29CD FD3B
Comment: Christoph Atteneder <christoph.atteneder@gmail.com>

xsFNBFl5pBEBEACmse+BgUYi+WLTHR4xDwFE5LyEIT3a5t+lGolO3cVkfw5RI+7g
FEpxXzWontiLxDdDi34nr1zXOIEjSgQ7HzdtnFiTRN4tIENCBul4YiCOiyBi5ofN
ejAHqmeiO0KsDBQZBdyiK1iWi6yNbpG/rARwHu/Rx5ouT1YX1hV92Qh1bnU+4j4O
FcePQRNl+4q/SrtKdm047Ikr/LBvy/WYBYe9BcQGhbHI4DrUOSnIuI/Zq7xLF8QS
U/an/d0ftbSBZNX3anDiZjzSmR16seRQtvRO6mehWFNlgLMOGgFeJzPkByTd1TlV
K/KaHKQ71FNkRiP87pwkHZI5zJPAQfve+KmYPwOyETUaX43XOuixqutUV6Lrd0ng
bKe6q4nZDOWi5a4I3+hkrfzaGOKm9TlZoEmpJHh6pa5ULoDnEpKCg5Dgn3NGwokw
57sDAC2mtf41/uSkR20ALN1q4iOLXiHn+T6Z+Uq7aL3OcKGcBu4xC6Jfofmmdfdd
QxEEaeHvAI9ETlKy3tsMhEs5XD6m90rCKLnb97Y8eT/xJL4/oDsxI0o7qICz1nFS
2IhV8xULZ2533vNQPMEbSLoTzgz1OEPYwI1b+YJDFlp1y0XRiEtDZiAFfgsJY7UE
DizfuUFsK5LOkw2+NVmLphDVrDW1MXbhX1xspZDmBG9giE08sPtHj/EZHwARAQAB
zTNDaHJpc3RvcGggQXR0ZW5lZGVyIDxjaHJpc3RvcGguYXR0ZW5lZGVyQGdtYWls
LmNvbT7CwZQEEwEKAD4CGwMFCwkIBwMFFQoJCAsFFgIDAQACHgECF4AWIQTLNtfS
67LjXZt1UAvNXcHFKc39OwUCYORPLgUJCy0SHAAKCRDNXcHFKc39O1MkD/95Qi1T
aBRhARPnxZgFWMfBIAtKyCC2AFMx28V+AQpAmWR/Stm+qGCAfV4HFTmekATzR4X4
L7E35AJxOHyhmLiEXZGw9jULRGhg2TqIJlDLClh+D2QxRcKcRnRcjNGV1tiBOKCP
M8KUytwMdEb7PLnQRBl2BV183g1LJ9OlLgSgPgk7sJjLLM6qobxJ3o6hpuKTW5X0
pS8qiFjQlv9AOYEllcBBWKzKgUUqKrCaxruIG761fQC4Fwx+FWB+Je48xM6nm8Vh
n4oyuC0+Gu8sglI/d/RI8e+gpMk5obWEAXDn5PFSsKaJnOW38ft0Qvq/3Ukg4AS0
3lXF9LXWxuv6e0gIlpt6HRW7L3pYVyqROVuGcgUWGf9tajjoAqtHHq5a467uj6bp
SfWC0bxUXrr4+x0/yROeq07yw8h9bUi9ygS+kvKYVrgjGQVIY3TvmY9UJD65L733
lb2DvllX+Dzxyt9aV/6F9i2lajOKonmLJmbZ25W6W7wrR+hnmuDNqZmPthjynh9z
3IEYNicFhAlX8lZaGEu50M/MrK85H8L/xBWqXzHFbaCS6jvZOabC8LaEjrR98WrZ
IBzVdoS4JlbwbeN4W3H4zOl7ITNPul1Xb7sbnOi33vn6HxW7vp0ArTIw7ldad456
zEdLbtUUmAEsS7dXIajCau0klhdqGHD9VXoRK8LBlAQTAQoAJwUCWXmkEQIbAwUJ
B4YfgAULCQgHAwUVCgkICwUWAgMBAAIeAQIXgAAhCRDNXcHFKc39OxYhBMs219Lr
suNdm3VQC81dwcUpzf07OOoP/iiS9zOgX6/SvZrjYxHgOsgUuR5mdrQSSNz4haSQ
uTATuR/t7pCPJ8s2IpdyYRKgnE1LKQRm25Km8G0kLktfVWARx8YUgJ+wfzBxQhZ5
WyhM0t3B8qfeZRjyyf8uRV150sXZQtRFxXLkox1kdu3AlBrh/tXYrsodnmaE+Xzd
u4OQeXo9BMV6AqfCeDLTIXB14xsuyor9Am3GPL2BGG6neRrDL5O1GSGvr1kzMyRN
X6eprBX1Bcx12EYTv0yovIkbVWlFHGjh8+5G9tunJFZ8yDtwo7BoO8tfQWGlnwuk
cNF6j3wn+z6CY4UcWwv/WVro8v6ZGjrTgaKviGRApIh7ClHGYDXVO8TMghNGuYzr
ya8yp2ujBCz3M08nlqmby9HSjYZZcOViSIc3+ZnBajm9TpTY1sQtQZznjZOdX/hQ
7Y8Tsqej+7i9HT8u4+EwO2h4EDSstlL8d/hLvRByi9k9+cVz75mtWVcm0JTASs+6
240yIik3+S6k7RchOCseiEtva8oMiT8c/Na+KW46BlCMS54cS/ui8MQj8Jm5CHxU
Q/oJPDg5oYGPeOw+lQTz66x+FPWIlCWe9fG97QiaLmyz6xlH2xZSmhgZaCvi3dLp
SRma0FsYit8tSjtQBfJNYHvbrSaMCwKvK2TlHUj2/3I6NzrLKaiUfTkEFzTtJ+li
92P2zsFNBFl5pBEBEADLdOm3qGHv75gFkfdcuQVCLtczZkD9jTQGdSYdSt6u2gFR
GpslZnReItiVRe3050Rlg4ioIxdn/SiQNpgvb+hCupeyiv+6EHxYMpqZLvIUi47a
f01iGWdQ/c9ehnaplmkRnTJKpo3+1qctugOUwrFGxr59bZvT/D8ajX4Ar9wmqRYT
3IiuKP3uIF2O7VG2DAuuQ5qtzqCVlswBqng5mx+FOQXKGQZPkhcpJxf+TVhrsxNM
pek0ZIBIc53sgXGldW5klXFgaTaM6MmQuPSYIK8BSRMXeUEBNOolm8fjpFh2LTmy
UikHd5YgtRHABcyNAaGUOWRPFaE16VUF0HLGLW1io+QUXohCKL4t//gpo8VBUjO3
43npUcFVnodPxZw+ysReS7IVA5a3PySEcSPUWGXG4aFwdVDHP+Ul+QE9PmvOqRms
u8hNCUBrWqdR4ucn9cgzCa9YvPbtS3yl2x29ozDs8og2EdpGjztCA6QyepIz3HxQ
HDbfxaScOVN5gSsvLR5SDKBXf+YUxQX/T564925l8dnUndWN5erEI4XHkiAC0CPe
UH12Oe2tOJh1ETvwgscMy5SniNUKe80/U4le6AcfObmNlba8T3T5jxxdMvRZ4StT
6tlQDed1hxivTBLsiFf1cXmze8NrQ4VYvUiNlJnRrg3ay5QgYU9FHgmLdoxcRQAR
AQABwsF8BBgBCgAmAhsMFiEEyzbX0uuy412bdVALzV3BxSnN/TsFAmDkTy8FCQst
Eh4ACgkQzV3BxSnN/TsByw//a2+f7tQHVA/RrOCzwen1W0aWf3Fwscbg1kmNk8qz
qtRE677wJt1sktPXEmuhY+wRDa9x07O+3hBYfQDv6sLFx4HDU2d0Lfthgirj6eFu
xQn9OTHIek24qpP6lAtfzvSGMV4EkJWUakqh6BisBPbbniGDzucs/5HkbSP4jvX4
Nh3vV8+8t6LvlSL0zrTat11zAVtwbLlDNWnmv0xrIDfcOdeN/j0LkALFY7UkKxwW
MXaOnpBk4RMRCmlVw9mTH36dxAVRo2t15crVGdBydH85UsNXb2r+FErZyHwbGWEw
WXXx2TCz5F4MS9A0GpatWxZskAo+BrLlAdktzIXQQpjQi1mZ1BfYk6IGJvTa6OSc
pQTjyT08QZHCPww/YnwYFFUwhp7/pMDGGZh9XISaPueiJNPERw7DVEYZysIoxp+W
r1jYzGFxaQfaAtZsQiCj+7F73eAF6kUyY0m2+Enp+hjZl9njDT+hBaZTTN1wetr3
bG2R2CDa+fnqShyYmY+HPAWXkJNaX2K8jvTQw+iljSM4D3G5TOCnXwmqiO3dT+MM
e/QG5g+n9ZM5jbbADyX8d2emXRy/rR7a6GijT479BGeZyY0EWcrVaF4ZrXdLYkLj
dVyaegYABm8PiBUnt9lSDx2WqwwoUemutxAIKmrggT84FAbfX1OMAw9ZTggYyNSR
vmnCwXwEGAEKAA8FAll5pBECGwwFCQeGH4AAIQkQzV3BxSnN/TsWIQTLNtfS67Lj
XZt1UAvNXcHFKc39OxuRD/92yDV+/NhkO/3z1T91y9kdZ1AeGNHPX5jahKZHOxpv
6VsIx32du0drgI2NzlTEWtmy3zft8G9Q+uV00UID3hQ4QQ5FFQme0t1weE2c8r6k
GQvMtxBu3DD2cP3gHqGp0s0PmJaGcq4jsFcWDy7Mfd9GsSHE0HJ9e/VTKr1qiWxu
9GN8Wwvuf/7TZoq4HMg7lVsV503RpnnaYDVqigMoPr2SmkFZez6zE+QeSkJ7ElSS
vd6SOwfyOTXQ696vwM2PxA3iKLYjf/GxEIK+u9wo1+bgtHj0on7lhRSQ6jXFZLA7
NYdX2WmfZpswSqoiqe5u8Q1yU2KohsTq+QBogvK5uKWDFNVtzpKe1RqtQ7/n2PZr
Rz6kOceAsHv31ueHSYCZdcJU7dnXgalofXmJpQKBGDHH24/6VAiPou2s86xzlvkn
pUuB3FCUNd6Mp709ozr5qC0gGZsjbi3fqWOZGuFrZAf3QyHIM+sn9TpyH+KWBNM1
FZt9OwAtHHM2ace+PRjnkix1iDdubvVBP33H2Q26jOjsjgysYSVL70/DLfWVxjXs
5LL8rXCqksJbV7FzhLAlGwAiY1ZAs1zAzHV8gGE+LK6FFWyOgIR1rms741gXxTy7
9SteLE1ewcYGBrKdY7+sj0oWDR81h44bxeebgLG5apS7QUMy/oGGQwJPPZfjFydu
0g==
=bAg3
-----END PGP PUBLIC KEY BLOCK-----

```

/cc https://github.com/bisq-network/compensation/issues/1237

**Comment by ripcurlx** [comment]
## Cycle 44 report

The release of [v1.9.9](https://github.com/bisq-network/bisq/releases/tag/v1.9.9) was the first public release of @alejandrogarcia83 🚀 . Congrats! It ships a couple of fixes for v1.9.7 to improve the new decentralization of fee distribution.


/cc https://github.com/bisq-network/compensation/issues/1237

**Comment by ripcurlx** [comment]
## Cycle 45 report

No release took place in the last cycle. I've revoked my maintainer bond now based on the agreement to phase out my maintainer role as @alejandrogarcia83 took over the release duties. Thanks for that! I'll still help out with reviewing and merging every now and then until my availability increases again to contribute more.

/cc https://github.com/bisq-network/compensation/issues/1251

**Comment by alvasw** [comment]
Hi!

I was the Bisq 2 repository maintainer for the past two years. The previous maintainer gave me maintainer rights before leaving the project, even though I never locked up a bond. Up until last week it didn't matter because Bisq 2 wasn't released yet and there was no risk involved. I think it's time to hand over my maintainer rights to @alejandrogarcia83 (bonded Bisq 2 Release Manager).

@gabernard Please remove me as a maintainer and give @alejandrogarcia83 (bonded Bisq 2 Release Manager) the needed permissions to manage the Bisq 2 repository.

I'll still contribute as usual and review PRs in the future.

**Comment by sqrrm** [comment]
I'm unlocking my bond for this role since I'm no longer active.

**Comment by pazza83** [comment]
@alejandrogarcia83 is this role still active?

**Comment by alejandrogarcia83** [comment]
> [@alejandrogarcia83](https://github.com/alejandrogarcia83) is this role still active?

Yes, my bond is still locked. I can post the updates soon. Sorry for the delay.
